client/src/cmdlfhitaghts.c

```diff
@@ -37,28 +37,45 @@
 static int CmdHelp(const char *Cmd);
 
 static const char *hts_get_type_str(uint32_t uid) {
+    // source 1: https://www.scorpio-lk.com/downloads/Tango/HITAG_Classification.pdf
+    // IDE Mark
+    // Each HITAG chip contains an unique Device Identifier (IDE ) so called a Serial Number.
```

Comment:
Unique Identifier  ( UID )  4 bytes.   its not IDE..

![image](https://github.com/user-attachments/assets/2c2c90b9-4b99-4e22-ad64-f7a5f691debc)

Reply:
Simply copy/paste from the unofficial HITAG_Classification.pdf. I'm confused about the strange abbreviation "IDE" too. I can't find anything on the internet about the "IDE mark" except in the PDF itself. I don't even know if it's really applicable to Hitag S.

Reply:
I never heard of IDE mark too.